sks/locals.tf

```diff
@@ -0,0 +1,15 @@
+locals {
+  metrics_storage = var.metrics_storage != null ? {
+    storage_config = {
+      type = "s3"
+      config = {
```

Comment:
The naming here (and the same in others variants) seems a little bit repetitive.
For exemple `local.metrics_storage.storage_config.config` seems redondent. Maybe something like `local.metrics.storage.config` is better (don't know if it's something to much spread in various DevOps Stack modules to be changed)

Reply:
I see your point. I remember last time around I decided to be more explicit on the naming because I think that the module interface to the outside needs to be clear what it does. For example, on the module call having a variable named just `metrics` could be confusing and would require consulting the documentation. The downside to this approach effectively is that redundancy on the locals inside the module, but that should be transparent to the user anyway.

Either way, just changing it here requires we change other modules as well, the naming scheme is similar.